local-release

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# The purpose of this script is to publish artifacts to ~/local-repo
+
+LOCAL_REPO="${HOME}/local-repo"
+
+if [ ! -d $LOCAL_REPO ]; then
+  # If this error is encountered, then (1) $HOME is not set correctly or (2) ~/local-repo doesn't exist
+  echo "Cannot perform local release, '$LOCAL_REPO' is not a valid directory"
+  exit 1
+fi
+
+VERSION=$(awk 'BEGIN { FS = "=" }; $1 == "version" { print $2 }' gradle.properties)
```

Comment:
Will this handle trailing whitespaces?

Reply:
It won't, fixed that in the latest diff and fixed it in the other files as well. What's interesting is that other Gradle tasks break if there's tailing whitespace, so it's a moot point.